src/title.jsx

```diff
@@ -0,0 +1,7 @@
+import React, { memo } from 'react';
+
+export function Title({ text }) {
+  return <h2>{text}</h2>;
+}
+
+export const MemoizedTitle = memo(Title);
```

Comment:
컴포넌트 이름에 memorized 라는 이름을 붙인 이유가 있을까요!? 
```suggestion
export default memo(Title)
```

Reply:
memo로 감싼 컴포넌트는 동일한 props를 전달해야 리렌더링을 하지 않고 최적화가 되기 때문에 혹시 해당 컴포넌트에
객체를 전달하게 되면 useMemo나 useCallback을 이용해서 전달해야 되기 때문에 눈에 띄기 위해 MemoizedTitle이라 이름을 붙였습니다.

Reply:
text 로 들어가는값은 항상 string 이니깐, Memorized 라는 접두사는 빼도 괜찮을거같아요 :)

Reply:
넵, 반영하겠습니다!